docs/pages/contributing/documentation/how-to-contribute.mdx

```diff
@@ -1,168 +1,306 @@
 ---
-title: How to Contribute to Teleport's Documentation
-description: Follow this guide to get started contributing changes to Teleport's documentation
+title: How to contribute to Teleport documentation
+description: Follow this guide to learn how to contribute changes to Teleport documentation.
 ---
 
-This guide describes the general workflow for making documentation changes.
-It assumes you already have Node >=14 and `yarn` installed.
+We encourage everyone with an interest in improving Teleport documentation to contribute content, submit issues, and suggest changes.
+This guide describes the process for making changes to the documentation whether you are an internal or external contributor.
 
-## Step 1/4 Set up your local environment
+## Before you begin
 
-Clone the `gravitational/docs` repository via `git clone`:
+Before you begin, verify that your environment meets the following prerequisites:
 
-```code
-$ git clone https://github.com/gravitational/docs
-```
+* You have a code editor, a GitHub account, and experience using command-line programs, including `git` commands and command line options.
```

Comment:
I'm not sure about this one. Since our audience is technical (https://goteleport.com/docs/contributing/documentation/style-guide/#voice), there's an argument to be made that expected docs contributors would have a code editor and experience using command-line programs. What do you think?

Reply:
In general, I'd prefer not making too many assumptions about the audience. A technical audience might skim over this or dive directly into forking the repo without reading any doc. If we expend the audience for the product to address the needs of businesses that might be less technically savvy, the doc should serve those people too.